improver/calibration/__init__.py

```diff
@@ -606,8 +606,8 @@ def add_static_feature_from_cube_to_df(
     """Add a static feature to the forecast DataFrame from a cube based on the
     feature configuration. Other features are expected to already be present in the
     forecast DataFrame. Columns that are float, after converting from a Cube to a
-    DataFrame, are rounded to a specified number of decimal places before merging
-    to avoid precision issues.
+    DataFrame, and within the list of column names provided for merging, are rounded
+    to a specified number of decimal places before merging to avoid precision issues.
```

Comment:
```suggestion
   forecast DataFrame. Columns within possible_merge_columns that are float after converting from    
   a Cube to a DataFrame, are rounded to a specified number of decimal places before merging to 
   avoid precision issues.
```

Is this wording a bit easier to parse?

Reply:
Updated as suggested.